skypy/power_spectrum/tests/test_class.py

```diff
@@ -25,22 +25,20 @@ def test_classy():
     '''
     from skypy.power_spectrum import classy
 
-    Pl15massless = Planck15.clone(name='Planck 15 massless neutrino', m_nu=[0., 0., 0.]*u.eV)
-
     redshift = [0.0, 1.0]
     wavenumber = np.logspace(-4.0, np.log10(2.0), 200)
-    pzk = classy(wavenumber, redshift, Pl15massless)
+    pzk = classy(wavenumber, redshift, Planck15)
```

Comment:
```suggestion
    pzk = classy(wavenumber, redshift, Pl15massless)
```
Don't make any changes to the class tests in this PR. Revert to using Pl15massless cosmology. It doesn't matter that this is not the same as what we test in camb